further_link/endpoint/upload.py

```diff
@@ -18,8 +26,17 @@ async def upload(request):
         if not directory_is_valid(directory):
             raise web.HTTPBadRequest()
 
+        # Upload to further directory
         await do_upload(directory, work_dir, user)
 
+        # Upload to projects directory to run using miniscreen app
+        if is_miniscreen_project(directory):
+            if user is None:
+                user = default_user()
+            user_home = path.expanduser(f"~{user}")
+            project_work_dir = f"{user_home}/Desktop/Projects"
```

Comment:
Also, the location is meant to be {user_home}/Desktop/Projects/**Further**. I think this is important so that we can keep them separate in the miniscreen menu, implement the 'delete all' option for them, etc.

Actually, to be honest at some point the plan was to store the projects separately for each Further username and I'm not sure now why that didn't make it into the final spec. Maybe we should rethink it with that feature as it's important for shared school devices, although changes will be needed on Further frontend to provide the username in that case...